OctoPipeline/octopipeline
# Feat/profile

### Summary
Profile UI

### Changes

 - notsignedin.tx : page that opens up when a person tries to open a page when they are not signed in. for example the account page.
 - profile page: signing out would redirect to the home page. the name and email are dynamic but the print logs arent.

### To Test
- see the profile page. Maybe try creating another account and see if it updates your name and email. 
- accessing the account page without signing in
